.bento/config.yml

```diff
@@ -1,35 +0,0 @@
-autorun:
-  block: true
-formatter:
-- clippy: {}
-tools:
-  bandit:
-    ignore:
-    - any-other-function-with-shell-equals-true
-    - assert-used
-    - hardcoded-bind-all-interfaces
-    - hardcoded-password-default
-    - hardcoded-password-funcarg
-    - hardcoded-password-string
-    - hardcoded-sql-expressions
-    - hardcoded-tmp-directory
-    - import-lxml
-    - import-pickle
-    - import-subprocess
-    - import-xml-expat
-    - import-xml-minidom
-    - import-xml-pulldom
-    - import-xml-sax
-    - import_xml-etree
-    - md5
-    - random
-    - start-process-with-no-shell
-    - start-process-with-partial-path
-    - subprocess-without-shell-equals-true
-    - try-except-continue
-    - urllib-urlopen
-    run: true
-  flake8:
```

Comment:
Bento flake8 also included flake8-json, and flake8-debugger is there anything in those worth keeping?

Reply:
`flake8-debugger` seems reasonable: https://pypi.org/project/flake8-debugger/, but I don't find it too useful. Thoughts?

`flake8-json` is just for getting JSON output, so we don't need it :+1:

Reply:
Hmm yeah doesn't seem too useful looking at it. At this point porting those rules could be a nice first issue on rule writing.